### PR TITLE
Server now declares Psr\Log\LoggerInterface instead of concrete class

### PR DESCRIPTION
**Pull request by cspray**
See https://github.com/amphp/aerys/issues/96

### CODE REVIEW

**Comment by kelunik** [comment]
There are a few more occurrences: https://github.com/amphp/aerys/search?utf8=%E2%9C%93&q=Logger

Could you fix them as well?

**Comment by cspray** [comment]
@kelunik oops, sorry about that. Not sure what I was thinking. Clearly I shouldn't code before I drink coffee. Will hit those in a few.

File: test/WebsocketParserTest.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 use Aerys\Websocket;
 use Aerys\Websocket\Code;
 use Aerys\Websocket\Rfc6455Endpoint;
+Use Psr\Log\LoggerInterface as PsrLogger;
```
Comment (cspray) [inline test/WebsocketParserTest.php]: Inappropriate capital 'U'

Comment (kelunik) [inline test/WebsocketParserTest.php]: Hm?

Comment (staabm) [inline test/WebsocketParserTest.php]: First char in the line

Comment (kelunik) [inline test/WebsocketParserTest.php]: Oh, missed that.

**Pull request merged**